test/e2eTests/tests/checkIn_test.js

```diff
@@ -11,6 +15,7 @@ let Patient1 = null;
 
 Before(async (I, Apis, Data) => {
   I.say(`${LOG_TAG} Creating patient one`);
+  //Person1 = await Apis.person.create(Data.patients.patient1);
```

Comment:
Doesn't seem like this is needed.

Reply:
This is an error on my part, this needs to be uncommented. I was testing and terminated the test with the patient still in the database.